package test_28;
import java.io.IOException;

public class test_28
{
	public static void main(String[] args)throws IOException
	{
		int i = 1;
		do
		{
			System.out.println("第"+i+"次的迴圈");
			i++;
		}while(i <= 5);
			
		System.out.println("迴圈結束");	
	}
}
